### DISCUSSION TITLE
I am struggling to get intellisense of my carbon components in react

### DISCUSSION THREAD

**Discussion by amosmachora** [carbon-design-system/carbon#18079] [Help]
I have `@carbon/react` installed on version "1.68.0". Normally on a component you expect that when you hit `ctrl + space` on vscode you get to see the component props. In my case, I do not. Any guide on how to do this? I am using typescript version `5.6.3`.  I even tried installing a random library I found on the package page on npmjs.org called `typescript-config-carbon` but that was a futile attempt.

**Comment by amosmachora** [comment]
a dump of my `tsconfig.json`.

```
{
  "compilerOptions": {
    "esModuleInterop": true,
    "module": "esnext",
    "allowSyntheticDefaultImports": true,
    "jsx": "react",
    "skipLibCheck": true,
    "moduleResolution": "node",
    "lib": [
      "dom",
      "es5",
      "scripthost",
      "es2015",
      "es2015.promise",
      "es2016.array.include",
      "es2018",
      "es2020",
      "es2022",
    ],
    "resolveJsonModule": true,
    "noEmit": true,
    "target": "esnext"
  }
}
```

**Comment by adamalston** [comment]
Hi, does this issue only occur with Carbon components or does it occur with other imports as well?

**Comment by amosmachora** [comment]
I figured out the issue it was my carbon version. Its basically not an issue.
